src/app/services/iou.service.ts

```diff
@@ -18,11 +18,11 @@ export class IouService {
   }
 
   getImages(server: Server): Observable<any> {
-    return this.httpServer.get<IouImage[]>(server, '/compute/iou/images') as Observable<IouImage[]>;
+    return this.httpServer.get<IouImage[]>(server, '/images') as Observable<IouImage[]>;
```

Comment:
We could optimize the API request here. The endpoint `/images` takes a parameter to only return a given type of image.

![Screenshot from 2022-04-28 22-25-52](https://user-images.githubusercontent.com/4752614/165788140-0ee6d8e5-3087-419a-8e5e-810c0f0dfbd2.png)

So depending on the appliance type this would give us:

`this.applianceToInstall.dynamips` => `/images?image_type=ios`
`this.applianceToInstall.qemu` => `/images?image_type=qemu`
`this.applianceToInstall.iou` => `/images?image_type=iou`

Reply:
I have add `/images?image_type="ios/qemu/iou"`  key with `/images` endpoint